invenio_app_rdm/config.py

```diff
@@ -1476,6 +1476,8 @@ def github_link_render(record):
 APP_RDM_SUBCOMMUNITIES_LABEL = "Subcommunities"
 """Label for the subcommunities in the community browse page."""
 
+APP_RDM_DEPOSIT_NG_FILES_UI = False
```

Comment:
for me a bool configuration variable should be called with suffix `_ENABLED`. if we want to use it as it is it should get a string where we have the option for two literals: `Literal["multipart", "singlepart"]`, so the idea is that we could add more options afterward. i am not sure if we need this possibility

Reply:
Thanks Christoph, good point, I'll add the suffix. This uploader implementation is meant to replace the current one, while supporting multiple transfer types (Uppy determines whether to use singlepart or multipart transfer per each individual file, depending on its fsize). As for the second approach, enumerating allowed transfer types (I'm also not sure if there's a case for prohibiting multipart uploads?), uploader UI has already knowledge of it from: https://github.com/inveniosoftware/invenio-app-rdm/pull/2994/files#diff-a779205cb118ac986e91089f0dcc0ed648e4b7a32b252151438c76a8fab7f10fR390-R391